skrub/_fuzzy_join.py

```diff
@@ -148,35 +148,35 @@ def fuzzy_join(
     ...     left_table,
     ...     right_table,
     ...     on="Country",
-    ...     suffix="_capitals",
+    ...     suffix="_right",
     ...     max_dist=1.0,
     ...     add_match_info=False,
     ... )
-      Country Country_capitals Capital_capitals
+      Country Country_right Capital_right
```

Comment:
This is an example where having a prefix or format string would be better than a suffix: 
`Country Capitals.Country Capitals.Capital` would be even clearer

Reply:
I guess we could make the same change in the `Joiner` docstring?
:thinking:  another option would be to change the column names so that we don't need the suffix in the toy example but it's probably useful to show it

Reply:
That's true it could be changed in the `Joiner` docstring too -> i'll do it
Changing the col names could also work. I'll just vouch for removing `Country_capitals` and `Capitals_capitals` because I find it misleading

Reply:
In the `fuzzy_join` example I'll suggest the suffix `_right`, and in the `Joiner` the suffix `_aux` just to match the class/function arguments

Reply:
Great, thanks